editor/editor_node.cpp

```diff
@@ -288,6 +288,7 @@ void EditorNode::_notification(int p_what) {
 
 		Engine::get_singleton()->set_editor_hint(true);
 
+		OS::get_singleton()->set_low_processor_usage_mode_sleep_usec(EDITOR_DEF("interface/editor/low_processor_mode_sleep_usec", 6900));
```

Comment:
Properties should only be defined once, so this should be `EDITOR_GET` as it's already defined in `editor_settings.cpp`.

Reply:
Same for the others. This also prevents hardcoding the default value several times.